pkg/loop/config.go

```diff
@@ -0,0 +1,35 @@
+package loop
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+var EnvPromPort = "CL_PROMETHEUS_PORT"
```

Comment:
My instinct was to expose this as mutable so that `package main`s can make their own decisions about what it should be, but maybe this is not necessary :thinking:

Reply:
the plugin can't choose the value - it must be guaranteed to be unique and only the core node has enough context to do that

Reply:
>  it must be guaranteed to be unique and only the core node has enough context to do that.

That's why I think it's important to be mutable. And I'm trying to avoid making assumptions about the plugin implementations from here. A practical case could be if we want to write a test host for some reason.

Reply:
Oh but we can choose to not inherit the host env with newer hashicorp, so maybe this is nbd.

Reply:
The name of the env var. If there was a collision to avoid, or a different standard to follow than `CL_` prefixing everything, then this would need to be mutable for that to be possible. But we don't want to inherit the host env when launching a plugin anyways, so collisions don't really matter in practice, and the `CL_` standard doesn't even need to apply here. Maybe we should actually considering using something else, e.g. `LOOPP_` :shrug: